### DISCUSSION TITLE
Is it normal that when running 0200 command it shows 16 bytes but 0218 shows 8 bytes?

### DISCUSSION THREAD

**Discussion by ej012345** [picocomputer/community#13] [Q&A] [answered]
![p-23 06 09-12](https://github.com/picocomputer/discussions/assets/131208326/0b24a483-c0c4-4ef2-97a7-a0e62bf2235f)
wondering if it's a wiring issue or working as intended...

**Comment by ixmeins** [comment, marked as answer]
Yes, presumably because you are half way through a 16 byte block as far as the monitor is concerned.
